frontend/src/components/account/Auth.tsx

```diff
@@ -23,7 +23,7 @@ function Auth() {
   const location = useLocation();
   const history = useHistory();
   const clusterConf = useClustersConf();
-  const {from = { pathname: '/' }} = location.state as ReactRouterLocationStateIface;
+  const {from = { pathname: '/login' }} = location.state as ReactRouterLocationStateIface;
```

Comment:
With this change, once we're authenticated, we're directed to this login page again after logging in.